imagery-cesium/plugins/draw-annotations.visualizer.ts

```diff
@@ -280,4 +306,44 @@ export class CesiumDrawAnnotationsVisualizer extends BaseImageryPlugin {
 		this.reset();
 		super.onDispose();
 	}
+
+	private activeShapePointsDistance = () => {
+		const [ center ] = this.activeShapePoints;
+		const end: Cartesian3 = this.activeShapePoints.length === 2 ? this.activeShapePoints[1] : center;
+		return Cartesian3.distance(center, end);
+	}
+
+	/**
+	 * 
+	 * @param radius in pixels
+	 * @param center in pixels
+	 * @param rotation in degrees - optional - default 0
+	 */
+	private createCircleGeometry(radius: number, center: Cartesian2, rotation = 0): Geometry {
+		rotation = rotation / 180 * Math.PI;
+		let n = 36 // n sampling angles
+		let coords = [];
+		const cartesia3 = [];
+
+		for (let i = 0; i <= n; i++) {
+		  // get the current angle
+		  const θ = Math.PI * 2 / n * i + rotation;
+
+		  // get the radius at that angle
+		  const r = Math.pow(radius, 2) / Math.sqrt(Math.pow(radius, 2) * Math.sin(θ) * Math.sin(θ) + Math.pow(radius, 2) * Math.cos(θ) *  Math.cos(θ));
```

Comment:
this calculation is relevant only for ellipse! I think we can remove it and use the radius that the function gets as input

Reply:
try to use http://turfjs.org/docs/#circle
or this http://turfjs.org/docs/#destination
for the calculation

add the methods to imagery\utils\geo